packages/tasks/src/model-libraries.ts

```diff
@@ -686,6 +686,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.whisperkit,
 		countDownloads: `path_filename:"model" AND path_extension:"mil" AND _exists_:"path_prefix"`,
 	},
+	"ssr-speech": {
```

Comment:
Can you move the `ssr-speech` entry up in the file? We try to keep alphabetical order in the libraries definition.